scripts/install-nodejs

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+export DEBIAN_FRONTEND=noninteractive
+
+pushd /tmp/
+
+# http://node-arm.herokuapp.com/
+/usr/bin/curl --location --remote-name --silent 'http://node-arm.herokuapp.com/node_latest_armhf.deb'
```

Comment:
Kinda annoying that latest isn't a reference to a static version, but whaddaya gonna do?